templates/cookies.twig

```diff
@@ -28,7 +28,7 @@
 			</div>
 			{% if show_settings and cookies.enable_reject_all_cookies %}
 				<button
-					class="btn reject-all-cookies"
+					class="btn btn-discrete reject-all-cookies"
```

Comment:
I instead turned this into a new class, to mirror the usage of `btn-secondary` on the other button.

Reply:
See https://github.com/greenpeace/planet4-master-theme/pull/1646#issuecomment-1063986146